src/main/java/dataLoader/jsonDataLoader.java

```diff
@@ -0,0 +1,40 @@
+package dataLoader;
+
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Simple Class that parses data set from a json file
+ * without knowing the files format
+ * & prints out the key value pairs.
+ */
+
+
+
+public class jsonDataLoader {
+
+    private String location = "where the file is located";
+    private File json = new File(location);
+
+    public void parseJsonData(String json) throws IOException {
```

Comment:
Ok we parse the file, but where do we store the data? Also I think is better if we call the input variable ```json``` sth like ```jsonFile``` or similar...something that denotes that we expect a file

Reply:
I think  this function the way it stands right now should be declared as ```static```

Reply:
Renamed the input variable to jsonFile. Also, refactored parseJsonData.
The data can be stored on a class we'll create perhaps?

Reply:
Yes exactly, this is the intention... but where is this class :) :)

Reply:
isn't this going to be the sample class?